.github/workflows/r_valgrind.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   test-r-valgrind:
     name: r-package (ubuntu-latest, R-devel, valgrind)
-    timeout-minutes: 180
+    timeout-minutes: 240
```

Comment:
Please do this in a separate PR.

Reply:
done: https://github.com/microsoft/LightGBM/pull/4773